01/implement-most-frequent-k-elements.instructor.md

```diff
@@ -0,0 +1,29 @@
+# Instructor: Implement Most Frequent K Elements
+
+An example of a working implementation:
+
+```python
+def top_k_frequent_elements(arr, k):
+    if len(arr) == 1: return [arr[0]]
+    
+    hash_table = {}
```

Comment:
Prefer a descriptive name like `counts` or `freqency_map` (what it's for) rather than `hash_map` (what it is).

Reply:
✅